dc.dev.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.6'
+
+services:
+  app: &app_base
+    build: ./
+    volumes:
+      - .:/app
+    environment:
+      ENSURE_DEPENDENCIES: 1
```

Comment:
A small nitpick, consider using docker-compose utilities like `healthcheck` ?

```
app:
  depends_on:
    postgres:
      condition: service_healthy
    redis:
      condition: service_healthy
      
postgres:
  healthcheck:
      test: pg_isready -U postgres -h 127.0.0.1
      interval: 5s
redis:
  healthcheck:
      test: redis-cli ping    
```

Reply:
`condition` in `depends_on` is deprecated

please, see https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on and https://docs.docker.com/compose/startup-order/